app/models/filter.rb

```diff
@@ -102,7 +102,7 @@ def to_label
 
   def resource_type
     type = @resource_type || filterings.first.try(:permission).try(:resource_type)
-    type.presence
+    type.presence || N_('(Miscellaneous)')
```

Comment:
This is used in an SQL statement later on. It feels incorrect to use a translatable string there.